### ISSUE TITLE
Archive this repository?

### ISSUE THREAD

**Issue by Trott** [nodejs/promises#32]
There is no Promises Working Group and it doesn't seem like there's going to be one any time soon based on activity. This repository seems pretty quiet and most Promises discussion now occurs in nodejs/node and elsewhere. Should we archive this repository to reduce fragmentation of conversation? Discussions will get more attention elsewhere anyway.

**Comment by benjamingr** [comment]
@Trott yes, this repo has served its purpose. It can be archived at this point.

**Comment by meaku** [comment]
I was trying to figure out the current state of Promises in Node recently and couldn't find an answer. How about adding some links to active discussions or at least the conclusion of the discussions that happend in this repo before archiving in the README?

**Comment by benjamingr** [comment]
@meaku sounds good to me
